domains/shatanik.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Shatanik's Developer Portfolio",
+  "owner": {
+    "username": "shatanikmahanty",
+    "email": "shatanikmahanty@gmail.com"
+  },
+  "record": {
+    "CNAME": "shatanikmahanty.github.io/shatanikmahanty"
```

Comment:
Invalid CNAME record, it should just be a hostname without the / and additional characters.

Reply:
Thanks for the review!